src/gui/widgets/text_box_base.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-	Copyright (C) 2008 - 2025
+	Copyright (C) 2008 - 2024
```

Comment:
We shouldn't have copyright changes in this PR.
```suggestion
	Copyright (C) 2008 - 2025
```